app/src/main/java/com/asfoundation/wallet/ui/iab/IabInteract.kt

```diff
@@ -15,6 +16,10 @@ class IabInteract(private val inAppPurchaseInteractor: InAppPurchaseInteractor,
     const val PRE_SELECTED_PAYMENT_METHOD_KEY = "PRE_SELECTED_PAYMENT_METHOD_KEY"
   }
 
+  fun showSupport() = Completable.fromAction {
+    supportInteractor.displayChatScreen()
+  }
```

Comment:
The completable fromAction should be on the presenter, but I also think you don't need this to be a completable